test/test_parse_es.py

```diff
@@ -84,6 +86,134 @@ def test_extract_number_es(self):
         self.assertEqual(extract_number("seis punto dos", lang='es'), 6.2)
         self.assertEqual(extract_numbers("un medio", lang='es'), [0.5])
+        self.assertEqual(extract_number("cuarto", lang='es'), 0.25)
+
+        self.assertEqual(extract_number("2.0", lang='es'), 2.0)
+        self.assertEqual(extract_number("1/4", lang='es'), 0.25)
+
+        self.assertEqual(extract_number("dos e media", lang='es'), 2.5)
+        self.assertEqual(extract_number(
+            "catorce e milésima", lang='es'), 14.001)
+
+        self.assertEqual(extract_number("dos punto cero dos", lang='es'), 2.02)
+
+    def test_isFraction_es(self):
+        self.assertEqual(isFractional_es("vigésimo"), 1.0 / 20)
+        self.assertEqual(isFractional_es("trigésimo"), 1.0 / 30)
+        self.assertEqual(isFractional_es("centésima"), 1.0 / 100)
+        self.assertEqual(isFractional_es("milésima"), 1.0 / 1000)
+
+
+class TestDatetime_es(unittest.TestCase):
+
+    def test_datetime_by_date_es(self):
+        # test currentDate==None
+        _now = datetime.now()
+        relative_year = _now.year if _now.day == 1 else (_now.year + 1)
+        self.assertEqual(extract_datetime_es("11 ene")[0],
+                         datetime(relative_year, 1, 11))
+
+        # test months
+
+        self.assertEqual(extract_datetime(
+            "11 ene", lang='es', anchorDate=datetime(1998, 1, 1))[0],
+            datetime(1998, 1, 11))
+        self.assertEqual(extract_datetime(
+            "11 feb", lang='es', anchorDate=datetime(1998, 2, 1))[0],
+            datetime(1998, 2, 11))
+        self.assertEqual(extract_datetime(
+            "11 mar", lang='es', anchorDate=datetime(1998, 3, 1))[0],
+            datetime(1998, 3, 11))
+        self.assertEqual(extract_datetime(
+            "11 abr", lang='es', anchorDate=datetime(1998, 4, 1))[0],
+            datetime(1998, 4, 11))
+        self.assertEqual(extract_datetime(
+            "11 may", lang='es', anchorDate=datetime(1998, 5, 1))[0],
+            datetime(1998, 5, 11))
+        # there is an issue with the months of june through september (below)
+        # hay un problema con las meses junio hasta septiembre (lea abajo)
+        self.assertEqual(extract_datetime(
+            "11 oct", lang='es', anchorDate=datetime(1998, 10, 1))[0],
+            datetime(1998, 10, 11))
+        self.assertEqual(extract_datetime(
+            "11 nov", lang='es', anchorDate=datetime(1998, 11, 1))[0],
+            datetime(1998, 11, 11))
+        self.assertEqual(extract_datetime(
+            "11 dic", lang='es', anchorDate=datetime(1998, 12, 1))[0],
+            datetime(1998, 12, 11))
+
+        self.assertEqual(extract_datetime("", lang='es'), None)
+
+    # TODO fix bug causing these tests to fail (MycroftAI/mycroft-core#2348)
+    #         reparar error de traducción preveniendo las funciones abajo de
+    #         retornar correctamente
+    #         (escribido con disculpas por un Inglés hablante)
```

Comment:
escrito  ;)

Reply:
well, i guess that's the most accurate comment I've ever written, then!

(Ake: comment reads, roughly, "fix translation error preventing below functions from returning correctly. written with apologies by an English speaker." except 'escribido' should read 'escrito')